openupgrade_scripts/scripts/portal/15.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,15 @@
+---Models in module 'portal'---
+---Fields in module 'portal'---
+portal       / adyen.account            / website_message_ids (one2many): DEL relation: mail.message
+portal       / crm.team.member          / website_message_ids (one2many): NEW relation: mail.message
+portal       / uom.category             / website_message_ids (one2many): NEW relation: mail.message
```

Comment:
I don't understand why this file contains this line. 
The original one doesn't contains line 5 https://github.com/OCA/OpenUpgrade/blob/15.0/openupgrade_scripts/scripts/portal/15.0.1.0/upgrade_analysis.txt